README.md

```diff
@@ -14,19 +16,38 @@ npm install
 ### Compile all contracts
 
 ```bash
+npm run template:process
```

Comment:
If this is generating bor genesis then it should not be should under **_Compile all contracts_**.
Should we move it under **_Start Matic child chain_**  or maybe an independent section altogether?

Reply:
We need to process all templates (when the next PR will come). We will also process contracts for bor/main chain id.